server/routes/referrals/completionDeadlineView.ts

```diff
@@ -0,0 +1,69 @@
+import CompletionDeadlinePresenter from './completionDeadlinePresenter'
+
+export default class CompletionDeadlineView {
+  constructor(private readonly presenter: CompletionDeadlinePresenter) {}
+
+  private get dateInputArgs(): Record<string, unknown> {
+    const errorMessage = this.presenter.errorMessage ? { text: this.presenter.errorMessage } : null
+
+    return {
+      id: 'completion-deadline',
+      namePrefix: 'completion-deadline',
+      fieldset: {
+        legend: {
+          text: this.presenter.title,
+          isPageHeading: true,
+          classes: 'govuk-fieldset__legend--xl',
```

Comment:
what's the reason for having css references in here?

Reply:
This class is a "view", so anything to do with rendering might appear here, including which CSS classes to use on a title.